src/Uno.UI/UI/Xaml/FrameworkElement.wasm.cs

```diff
@@ -274,6 +274,29 @@ public double Width
 		}
 		#endregion
 
+		#region FlowDirection Dependency Property
+		[GeneratedDependencyProperty(DefaultValue = FlowDirection.LeftToRight, ChangedCallback = true)]
+		public static DependencyProperty FlowDirectionProperty { get; } = CreateFlowDirectionProperty();
+
+		public FlowDirection FlowDirection
+		{
+			get
+			{
+				var value = ReadLocalValue(FlowDirectionProperty);
+				if (value == DependencyProperty.UnsetValue)
+				{
+					return (Parent as FrameworkElement)?.FlowDirection ?? FlowDirection.LeftToRight;
+				}
+
+				return (FlowDirection)value;
+
+				// Workaround source generator error when this statement is missing.
+				_ = GetFlowDirectionValue();
```

Comment:
Workaround the following:

https://github.com/unoplatform/uno/blob/8ba96564207f22e88f05e9c8cfe54d317deafe2e/src/SourceGenerators/Uno.UI.SourceGenerators/DependencyObject/DependencyPropertyGenerator.cs#L349

Any concerns? @MartinZikmund @jeromelaban